src/js/Controllers/UIController.js

```diff
@@ -147,6 +150,22 @@ class UIController {
                 }
 
                 return null
+            case "CancelInteraction":
+
+                const state2 = store.getState()
+                var app = state2.ui[state2.activeApp]
+    
+                if (rpc.params.functionID === 10 && app.isPerformingInteraction
+                     && (rpc.params.cancelID === -1 || rpc.params.cancelID === app.interactionCancelId)) {
```

Comment:
-1 should not be used to indicate that a cancelID was not provided. It would just be omitted from the message in this case, so the expression would be `rpc.params.cancelID === undefined`.

Reply:
fixed in https://github.com/smartdevicelink/generic_hmi/pull/176/commits/930b56099e0354ed4858bbeb55c2bd1a43be7ced